### DISCUSSION TITLE
Disable a specific tab using CTkTabView

### DISCUSSION THREAD

**Discussion by NanoParticles** [TomSchimansky/CustomTkinter#1926] [Q&A] [answered]
Is there some way to disable a specific tab after adding the tabs in?

**Comment by IfTrueEqualsEqualsTrue** [comment]
You can try to acces the CtkSegmentedButton._buttons_dict which contains the buttons used by the widget to build itself. You can use these buttons as normal buttons and use classic methods on them, like configure(state='disabled'). It might work !

**Reply by NanoParticles** [reply]
Example? Im having trouble understanding how to get there.

Im coming up with this:

```
root = CTk()
rootTabs = CTkTabview(root)
rootTabs.grid(row=0,column=0)
rootTabs.add("Example Tab")
rootTabs.add("Example Tab 2")

rootTabs._segmented_button._buttons_dict.get("Example Tab").configure(state="disabled")
```

It doesn't change the state. Unless Im doing it wrong

**Comment by Akascape** [comment, marked as answer]
```python
tabview._segmented_button._buttons_dict["Tab_name"].configure(state="disabled")
```
This method works for me, make sure to change the tab before disabling it (using `.set` method)!

**Reply by NanoParticles** [reply]
Thank you so much for point it out! I was doing it wrong then.
